code/modules/paperwork/paper_cutter.dm

```diff
@@ -0,0 +1,125 @@
+/obj/item/weapon/papercutter
+	name = "paper cutter"
+	desc = "Standard office equipment. Precisely cuts paper using a large blade."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "papercutter-cutter"
+	force = 5
+	throwforce = 5
+	w_class = 3
+	var/obj/item/weapon/paper/storedpaper = null
+	var/obj/item/weapon/hatchet/cutterblade/storedcutter = null
+	var/cuttersecured = TRUE
+	pass_flags = PASSTABLE
+
+
+/obj/item/weapon/papercutter/New()
+	storedcutter = new /obj/item/weapon/hatchet/cutterblade(src)
+
+
+/obj/item/weapon/papercutter/suicide_act(mob/user)
+	if(storedcutter)
+		user.visible_message("<span class='suicide'>[user] is beheading \himself with [src.name]! It looks like \he's trying to commit suicide.</span>")
+		playsound(loc, 'sound/weapons/bladeslice.ogg', 50, 1, -1)
+		return (BRUTELOSS)
+	else
+		user.visible_message("<span class='suicide'>[user] repeatedly bashes [src.name] against \his head! It looks like \he's trying to commit suicide.</span>")
+		playsound(loc, 'sound/items/gavel.ogg', 50, 1, -1)
+		return (BRUTELOSS)
+
+
+/obj/item/weapon/papercutter/update_icon()
+	overlays.Cut()
+	icon_state = (storedcutter ? "papercutter-cutter" : "papercutter")
+	if(storedpaper)
+		overlays += "paper"
+
+
+/obj/item/weapon/papercutter/attackby(obj/item/P, mob/user, params)
+	if(istype(P, /obj/item/weapon/paper) && !storedpaper)
+		if(!user.drop_item())
+			return
+		playsound(loc, "pageturn", 60, 1)
+		user << "<span class='notice'>You place \the [P] in [src].</span>"
+		P.loc = src
+		storedpaper = P
+		update_icon()
+		return
+	if(istype(P, /obj/item/weapon/hatchet/cutterblade) && !storedcutter)
+		if(!user.drop_item())
+			return
+		user << "<span class='notice'>You replace [src]'s [P].</span>"
+		P.loc = src
+		storedcutter = P
+		update_icon()
+		return
+	if(istype(P, /obj/item/weapon/screwdriver) && storedcutter)
+		playsound(src, 'sound/items/Screwdriver.ogg', 50, 1)
+		user << "<span class='notice'>\The [storedcutter] has been [cuttersecured ? "unsecured" : "secured"].</span>"
+		cuttersecured = !cuttersecured
+		return
+	..()
+
+
+/obj/item/weapon/papercutter/attack_hand(mob/user)
+	src.add_fingerprint(user)
```

Comment:
you can't call parent here because otherwise you pick it up BECAUSE IT'S AN ITEM